var/spack/repos/builtin/packages/py-pytaridx/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPytaridx(PythonPackage):
+    """Python module/library for creating and maintaining a rapidly searchable
+       index for a tar-file. This allows "direct access" of members (files) in
+       the tar archive."""
+
+    homepage = "https://github.com/LLNL/pytaridx"
+    git      = "git@github.com:LLNL/pytaridx.git"
```

Comment:
```suggestion
    git      = "https://github.com/LLNL/pytaridx.git"
```
I've had a lot of issues with `git@github.com:` when SSH keys aren't configured correctly. Downloading over https should be safer.